src/cljc/ataru/schema/form_schema.cljc

```diff
@@ -224,6 +223,16 @@
    (s/optional-key :tarjonta)           FormTarjontaMetadata
    (s/optional-key :person-oid)         (s/maybe s/Str)})
 
+(s/defschema Person
+  {:oid         (s/maybe s/Str)
+   :turvakielto s/Bool
+   :yksiloity   s/Bool})
+
+(s/defschema VirkailijaApplicationReview
+  (-> Application
```

Comment:
Someone comment is this ok? Will the metadatas containing schema names break or something? Seems to work though..

Reply:
It should be in this case OK. Generally speaking you should use `schema-tools.core` functions to eg. dissoc keys from schema. I changed the code to create this schema to use `schema-tools.core` as an example.